.github/workflows/config/config.json

```diff
@@ -4,6 +4,7 @@
 	"MD046": { "style": "fenced" },
 	"MD029": { "style": "one" },
 	"MD014": false,
+	"MD040": false,
```

Comment:
I think this it's in this PR [1] also, so please rebase after merging that.

[1] https://github.com/unikraft/docs/pull/139